cdc/model/owner.go

```diff
@@ -208,12 +212,12 @@ func (i *TableReplicaInfo) Clone() *TableReplicaInfo {
 	return &clone
 }
 
-// TaskStatus records the task information of a capture
+// TaskStatus records the task information of a capture.
+//
+// Deprecated: only used in API. TODO: remove API usage.
```

Comment:
It looks the whole `TaskStatus` is labeled as `deprecated`.

It has 3 different usage:
1. for old scheduler to track scheduling information.
2. for new ticdc to keep compatible with the old version.
3. for OpenAPI, return table's information.

I think the first one usage is deprecated, the last 2 is still useful.

Reply:
> 2. for new ticdc to keep compatible with the old version.

I don't think so, we only need to decode keys left by old version, not values.

> 3. for OpenAPI, return table's information.

We will replace them with other structs defined in http_model.go later.